stat_plan9.go

```diff
@@ -41,6 +41,11 @@ func translateSyscallError(err error) (uint32, bool) {
 	return 0, false
 }
 
+// isRegular returns true if the mode describes a regular file.
+func isRegular(mode uint32) bool {
+	return mode&S_IFMT == syscall.S_IFREG
+}
+
```

Comment:
This check is not _really_ OS dependent, as it works through the internals of SSH flags. With the `filexfer` this can be done in a sufficiently generic way. I just put this here, as a convenience before the other filexfer implementation.